assets/fonts/icon_preview.html

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
@JeyKeu Could you remove this file and fix `.gitignore` to account for the move?